platform/security/certificate/security-certificate-keystoreeditor/src/main/java/org/codice/ddf/security/certificate/keystore/editor/KeystoreEditor.java

```diff
@@ -112,15 +114,15 @@ private void addProvider() {
 
     private void init() {
         try {
-            keyStore = KeyStore.getInstance(System.getProperty("javax.net.ssl.keyStoreType"));
-            trustStore = KeyStore.getInstance(System.getProperty("javax.net.ssl.keyStoreType"));
```

Comment:
This wasn't a but necessarily. It just made the keystore and truststore be the same filetype. This won't break anything, but allow us to have different types in the future.